log/grpc.go

```diff
@@ -1,12 +0,0 @@
-package log
-
-import (
-	"io/ioutil"
-	"log"
-
-	"google.golang.org/grpc/grpclog"
-)
-
-func init() {
-	grpclog.SetLogger(log.New(ioutil.Discard, "", log.LstdFlags))
```

Comment:
surprised that this is in the make it smaller/faster PR.  Is it because it should've been to SetLoggerV2?

Reply:
nvm we're backlvl with v2.

Reply:
These are left the to main of the binaries and just remove a dep